<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Separatore</class>
 <widget class="QMainWindow" name="Separatore">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>210</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Separatore</string>
  </property>
  <property name="windowIcon">
   <iconset resource="risorse.qrc">
    <normaloff>:/resources/icon_mmas.png</normaloff>:/resources/icon_mmas.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">/* sets background color  */
QWidget {
background-color:rgba(255,255,255,0);
}
QWidget#centralWidget{
	/*background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(217, 239, 255, 255), stop:0.5 rgba(239, 248, 255, 255), stop:1 rgba(210, 236, 255, 255));
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(223, 246, 255, 255), stop:0.481132 rgba(243, 243, 243, 255), stop:0.995261 rgba(223, 246, 255, 255));*/
	
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 232, 243, 255), stop:0.35545 rgba(247, 247, 247, 255), stop:0.635071 rgba(247, 247, 247, 255), stop:0.933649 rgba(220, 232, 243, 255));
	
}
QLineEdit {
     border: 1px inset rgb(188, 189, 207);
     border-radius: 5px;
     padding: 0 3px;
	/*background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0.127962 rgba(239, 251, 254, 255), stop:0.862559 rgba(219, 230, 239, 255));*/
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0.0545455 rgba(237, 240, 243, 255), stop:0.35545 rgba(247, 247, 247, 255), stop:0.635071 rgba(247, 247, 247, 255), stop:0.933649 rgba(237, 240, 243, 255));
     font: 11pt &quot;Arial&quot;;

 }</string>
   </property>
   <widget class="QRadioButton" name="radioButton_tab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>117</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Tabulazione</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_vir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>117</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Virgola</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_altro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>117</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Altro</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_sep">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="OKButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/* pushbutton style */
QPushButton {
     border: 1px groove rgba(255,255,255,0);
     border-radius: 5px;
	background-color: rgba(255, 255, 255,0);
	
	image: url(:/resources/filter_ok_not_hov.png);

 }

 QPushButton:hover {
     border: 1px outset rgb(188, 189, 207);
border-radius:3px;
	background-color: rgba(255, 255, 255, 0);
	
	image: url(:/resources/filter_ok.png);
 } 
QPushButton:pressed {
     border: 3px inset rgb(188, 189, 207);
	border-radius:3px;
 } </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="risorse.qrc"/>
 </resources>
 <connections/>
</ui>
